src/rard/research/models/citing_work.py

```diff
@@ -23,6 +23,10 @@ class Meta:
 
     ANONYMOUS_ORDERNAME = "anonymous_citing_author_order_name"
 
+    bibliography_items = models.ManyToManyField(
+        "BibliographyItem", related_query_name="bibliography_items", blank=True
```

Comment:
```suggestion
        "BibliographyItem", related_name="citing_authors", blank=True
```
* Use related_name rather than related_query_name
* Name should make sense from context of a bibliography item working backwards

See https://docs.djangoproject.com/en/4.1/topics/db/queries/#backwards-related-objects

Reply:
Ta